### PR TITLE
docs(security): avoid stacktraces in errors on container level

### PR DESCRIPTION
**Pull request by tasso94**
Related to CAM-11502

### CODE REVIEW

File: content/user-guide/security.md [hunk 1]
```diff
@@ -247,10 +247,55 @@ The container provides the session cookie. Please consult the documentation abou
...
+
+Please find below external documentation on how to configure your application server correctly:
+
+* Tomcat 7.0+
+    * Official Documentation
+        * [Security Considerations](https://tomcat.apache.org/tomcat-7.0-doc/security-howto.html#Valves)
+        * [Error Reporter Valve](https://tomcat.apache.org/tomcat-7.0-doc/config/valve.html#Error_Report_Valve)
+    * Alternative Resources
+        * [Securing Tomcat](https://wiki.owasp.org/index.php/Securing_tomcat)
+* Wildfly 8.2+: Official Documentation
```
Comment (tasso94) [inline content/user-guide/security.md:286]:
```suggestion
* Wildfly 10.1+: Official Documentation
```

**Review by yanavasileva — APPROVED** [review]
Consider adjusting the WildFly and JBoss points as on 7.12 and 7.13 upon merge.

File: content/user-guide/security.md [hunk 2]
```diff
@@ -247,10 +247,55 @@ The container provides the session cookie. Please consult the documentation abou
...
+    * [Servlet Container Configuration](https://docs.jboss.org/author/display/WFLY8/Undertow%20subsystem%20configuration.html#66322701_Undertowsubsystemconfiguration-Servletcontainerconfiguration)
```
Comment (tasso94) [inline content/user-guide/security.md:287]:
```suggestion
    * [Servlet Container Configuration](https://docs.jboss.org/author/display/WFLY10/Undertow%20subsystem%20configuration.html#91947134_Undertowsubsystemconfiguration-Servletcontainerconfiguration)
```

File: content/user-guide/security.md [hunk 3]
```diff
@@ -247,10 +247,55 @@ The container provides the session cookie. Please consult the documentation abou
...
+    * [Model Reference](https://wildscribe.github.io/WildFly/8.2/subsystem/undertow/servlet-container/index.html#attr-stack-trace-on-error)
```
Comment (tasso94) [inline content/user-guide/security.md:288]:
```suggestion
    * [Model Reference](https://wildscribe.github.io/WildFly/10.1/subsystem/undertow/servlet-container/index.html#attr-stack-trace-on-error)
```

File: content/user-guide/security.md [hunk 4]
```diff
@@ -247,10 +247,55 @@ The container provides the session cookie. Please consult the documentation abou
...
+* JBoss EAP 7.0+: Official Documentation
+    * [Servlet Container Configuration](https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.0/html/configuration_guide/reference_material#idm139812627222560)
+    * [Model Refernce](https://wildscribe.github.io/JBoss%20EAP/7.0/subsystem/undertow/servlet-container/index.html#attr-stack-trace-on-error)
+* JBoss EAP 6.4: Official Documentation
+    * [Model Reference](https://wildscribe.github.io/JBoss%20EAP/6.4/subsystem/web/configuration/jsp-configuration/index.html#attr-display-source-fragment)
+* JBoss AS 7.1: Official Documentation
+    * [Web Subsystem: Disable stacktrace in response body](https://docs.jboss.org/author/display/AS72/Hardening%20Guidelines.html#76154434_HardeningGuidelines-Disablestacktraceinresponsebody)
+    * [Model Refernce](https://wildscribe.github.io/JBoss%20AS7/7.1.1/subsystem/web/configuration/jsp-configuration/index.html#attr-display-source-fragment)
```
Comment (tasso94) [inline content/user-guide/security.md:296]:
```suggestion
```

File: content/user-guide/security.md [hunk 5]
```diff
@@ -247,10 +247,55 @@ The container provides the session cookie. Please consult the documentation abou
...
+* Camunda Run/Spring Boot 2.1+
+    * Official Documentation
+        * [Javadocs about ErrorProperties.IncludeStacktrace](https://docs.spring.io/spring-boot/docs/2.1.0.RELEASE/api/org/springframework/boot/autoconfigure/web/ErrorProperties.IncludeStacktrace.html)
+    * Alternative Resources
+        * [Error Handling on Baeldung](https://www.baeldung.com/spring-boot-configure-tomcat#2-error-handling)
```
Comment (tasso94) [inline content/user-guide/security.md:301]:
```suggestion
* Camunda Run/Spring Boot 2.3+
    * Official Documentation
        * [Javadocs about ErrorProperties.IncludeStacktrace](https://docs.spring.io/spring-boot/docs/2.3.0.RELEASE/api/org/springframework/boot/autoconfigure/web/ErrorProperties.IncludeStacktrace.html)
    * Alternative Resources
        * [Error Handling on Baeldung](https://www.baeldung.com/spring-boot-configure-tomcat#2-error-handling)
```

**Pull request merged**